spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/DirectMessageListenerContainer.java

```diff
@@ -212,18 +206,6 @@ public void addQueueNames(String... queueNames) {
 		super.addQueueNames(queueNames);
 	}
 
-	@Override
-	public void addQueues(Queue... queues) {
-		try {
-			addQueues(Arrays.stream(queues)
```

Comment:
There are 3 methods - set/add/remove, and those with `Queue...` delegate to ones with `String...`, which actually do all the work.
IMHO methods with `Queue...` can even be declared final.